tools/gulptasks/scripts-css.js

```diff
@@ -92,6 +92,29 @@ function copyCSS(config) {
     );
 }
 
+/**
+ * Changes the product version in the CSS files.
+ *
+ * @param  {string} version
+ * Version to replace.
+ *
+ * @param  {string} folder
+ * Folder to replace the version in.
+ */
+function replaceVersionInFile(version, folder) {
```

Comment:
During regular `gulp scripts` there is no need to specify --release argv, but in this case it is not required to be here, so it can be aborted. In the case of `gulp dist` it will always be defined.

```suggestion
function replaceVersionInFile(version, folder) {
    if (!version) {
        return;
    }
```